### PR TITLE
XmlAuditfileFinancieel syntax fix

### PR DESCRIPTION
**Pull request by CGenie**
Some leftover + from git merging was in the file which resulted in
validation errors.

### CODE REVIEW

**Review by astirpe — APPROVED** [review]

File: l10n_nl_xaf_auditfile_export/views/templates.xml [hunk 1]
```diff
@@ -148,7 +148,7 @@
                         <docRef><t t-esc="l.ref" t-esc-options='{"widget": "auditfile.string999"}' /></docRef>
                         <effDate><t t-esc="l.date" /></effDate>
                         <desc><t t-esc="l.name" /></desc>
-                        <amnt><t t-esc="l.credit or l.debit" /></amnt>
+                        <amnt><t t-esc="round(l.credit or l.debit, 2)" /></amnt>
```
Comment (astirpe) [inline l10n_nl_xaf_auditfile_export/views/templates.xml]: I think you don't need to round here. The risk is that the rounding will generate an amount different than `l.balance`. See https://github.com/OCA/l10n-netherlands/pull/130#issuecomment-353287014

**Review by astirpe — CHANGES_REQUESTED** [review]

File: l10n_nl_xaf_auditfile_export/views/templates.xml (hunk 1 above)
Comment (CGenie) [reply l10n_nl_xaf_auditfile_export/views/templates.xml]: Well we had this error:
```
:507419:0:ERROR:SCHEMASV:SCHEMAV_CVC_FRACTIONDIGITS_VALID: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '0.00999999999999' has more fractional digits than are allowed ('2').
```

Comment (astirpe) [reply l10n_nl_xaf_auditfile_export/views/templates.xml]: Isn't abs(l.balance) fixing that issue as well?

Comment (CGenie) [reply l10n_nl_xaf_auditfile_export/views/templates.xml]: Yes, it works, thanks.

**Review by astirpe — APPROVED** [review]

**Comment by hbrunn** [comment]
please squash your commits to one about the country, one about the rounding

**Review by hbrunn — APPROVED** [review]

**Comment by CGenie** [comment]
There need to be 3 commits as they fix 3 different things: syntax error in xsd, JE country and rounding error.

**Comment by hbrunn** [comment]
ah right, the -+ line fooled me. Will merge after green runbot

**Pull request merged**